build/build.go

```diff
@@ -575,7 +575,7 @@ func NewMappingCallback(m *sourcemap.Map, goroot, gopath string) func(generatedL
 	}
 }
 
-func jsFilesFromDir(dir string) ([]string, error) {
+func JsFilesFromDir(dir string) ([]string, error) {
```

Comment:
We can ask for @neelance's opinion on this.

But my understanding is that GopherJS was initially an experimental project, back when it wasn't known if it'd be possible to support of all Go's features and have good performance, so focusing on functionality over documentation made sense since @neelance was the only developer.

Now that GopherJS has implemented the entire Go spec and has proven to be quite mature, reliable, performant, used in some production environments, and is getting more contributions from external people, I think it's a good idea to start caring about internal code quality, documentation more than in the past.

So I think it's a good idea to apply best practices for the new code being added, and also slowly work on improving existing code. Does that seem like the best way to go, or not?